### DISCUSSION TITLE
MinIO in Docker and environment variables for systemd

### DISCUSSION THREAD

**Discussion by dominiqueburow** [minio/minio#13683] [Q&A]
Hey everyone,

I am playing with MinIO for a while now, mostly in Docker via docker-compose. It runs nice with 4 nodes, 2 HDDs each. But every 12 hours, the system load explodes, rendering the whole cluster unusable for at least 10 minutes.

Interestingly, this is independent from the traffic at this time. It does happen every 12 hours, not matter how many object are pulled or pushed.

I found some articles describing that in deployments directly on Debian for example, the service file for MinIO can be customized to change for example TasksMax or LimitNOFILE. But how to accomplish this in Docker?

I tried to pass this as an environment variable in my docker-compose file, without success. Is it irrelevant, because there are no limits inside the Docker container, am I missing something or is it just not possible?

Thank you for your time and cheers,
Dominique

**Comment by harshavardhana** [comment]
@dominiqueburow looks like your setup has configuration issues and perhaps requires an architecture overview.

Architecture reviews are available with the help of our engineers via subscription - if this is a production use case you are free to consider this as an option.
